tests/conftest.py

```diff
@@ -330,6 +330,17 @@ async def do_clean_hard(postgresql_client):
     assert not postgresql_client.is_in_transaction()
     await postgresql_client.reload_schema_state()
     tables_in_db = await postgresql_client.fetch("SELECT table_name FROM information_schema.tables WHERE table_schema='public'")
```

Comment:
I would move this line further down so the tables and function stuff is grouped together. Now it's a bit of a weird order: get tables -> get functions -> drop functions -> drop tables.